src/core/Select/Select.tsx

```diff
@@ -109,7 +113,18 @@ export type SelectProps<T> = {
    */
   popoverProps?: Omit<PopoverProps, 'onShow' | 'onHide' | 'disabled'>;
 } & Pick<PopoverProps, 'onShow' | 'onHide'> &
-  CommonProps;
+  CommonProps &
+  Omit<
+    React.HTMLAttributes<HTMLDivElement>,
```

Comment:
Better to use `React.ComponentPropsWithoutRef<'div'>`

Reply:
Or maybe even `React.ComponentPropsWithoutRef<'select'>`

Reply:
Select no, I can change to div..
Whats the diff again?

Reply:
Well, for one, you can be explicit about ref.

Second, it's easier to read/remember I guess